SabreTools.Serialization/Readers/InstallShieldExecutableFile.cs

```diff
@@ -0,0 +1,60 @@
+using System.IO;
+using SabreTools.Data.Models.InstallShieldExecutable;
+using SabreTools.IO.Extensions;
+
+namespace SabreTools.Serialization.Readers
+{
+    public class InstallShieldExecutableFile : BaseBinaryReader<ExtractableFile>
+    {
+        public override ExtractableFile? Deserialize(Stream? data)
+        {
+            // If the data is invalid
+            if (data == null || !data.CanRead)
+                return null;
+
+            try
+            {
+                // Cache the initial offset
+                long initialOffset = data.Position;
+
+                // Try to parse the header
+                var header = ParseExtractableFileHeader(data);
+                if (header == null)
+                    return null;
+
+                return header;
+            }
+            catch
+            {
+                // Ignore the actual error
+                return null;
+            }        
+        }
+        
+        private static ExtractableFile? ParseExtractableFileHeader(Stream? data)
```

Comment:
2 comments:
- This can be `public static`
- Please copy-paste the description from an existing `ParseX` method and change the types mentioned